AbletonMCP_Remote_Script/__init__.py

```
@@ -352,8 +352,31 @@ def _get_session_info(self):
+                # playhead without polling separately. All values are best-
+                # effort: wrapped in try/except so older Live versions or
+                # unusual session states never break the existing response
+                # shape.
+                "is_playing": False,
+                "current_song_time": 0.0,
+                "song_length": 0.0,
+                "loop": False,
+                "loop_start": 0.0,
+                "loop_length": 0.0,
             }
+            try: result["is_playing"] = bool(self._song.is_playing)
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)